lib/virtual-net/Cargo.toml

```diff
@@ -14,7 +14,7 @@ thiserror = "1"
 bytes = "1.1"
 async-trait = { version = "^0.1" }
 tracing = "0.1"
-tokio = { version = "1", features = [ "sync", "macros", "io-util", "signal" ], default_features = false, optional = true }
+tokio = { version = "1", features = [ "sync", "macros", "io-util", "signal", "net" ], default_features = false, optional = true }
```

Comment:
The `net` feature is not available in all environments.

This would cause difficulties for the browser builds, I think, and wouldn't be good for users that want another network stack, or no network stack.

We should instead make the `host-net` feature enable `tokio/net`.

Reply:
@theduke I don't think that makes a difference because the `tokio` dependency is only included when `host-net` is enabled.

Reply:
Right now you are unconditionally enabling the net feature, which is not what we want.